content/sensu-go/6.1/web-ui/search.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "Search in the web UI"
+linkTitle: "Search in the Web UI"
+description: "The Sensu web UI supports searching the Events, Entities, Silences, Checks, Handlers, Filters, and Mutators pages. Learn more about searches in the Sensu web UI."
+weight: 30
+version: "6.1"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-6.1:
+    parent: web-ui
+---
+
+The Sensu web UI includes basic search functions you can use to build customized views of your Sensu resources.
+Sensu also supports advanced web UI searches based on a wider range of resource attributes and custom labels as a [commercial feature][1].
+
```

Comment:
Is this still true? If the web UI is now just part of the commercial package, do we just want a note at the top of every web UI page that matches the notes we use for other commercial features?

For example:
`COMMERCIAL FEATURE: Access [X FEATURE] in the packaged Sensu Go distribution. For more information, see Get started with commercial features.`